stdlib/public/runtime/LibPrespecialized.cpp

```diff
@@ -186,12 +182,16 @@ void _swift_validatePrespecializedMetadata(unsigned *outValidated,
               "Prespecializations library validation: unable to build metadata "
               "for mangled name '%s'\n",
               mangledName);
-      if (outFailed)
-        (*outFailed)++;
+      failed++;
+      continue;
     }
 
     if (!compareGenericMetadata(result.getType().getMetadata(), element.value))
-      if (outFailed)
-        (*outFailed)++;
+      failed++;
   }
+
+  fprintf(stderr,
+          "Prespecializations library validation: validated %u entries, %u "
+          "failures.\n",
+          validated, failed);
```

Comment:
Did you mean to leave this debug output in?

Reply:
Yes, this is meant to be called manually from a debugger now.